azure-fsharp/Program.fs

```diff
@@ -21,7 +22,7 @@ let infra () =
     let primaryKey =
         ListStorageAccountKeysInvokeArgs(ResourceGroupName = resourceGroup.Name, AccountName = storageAccount.Name)
         |> ListStorageAccountKeys.Invoke
-        |> Outputs.apply (fun storageKeys -> storageKeys.Keys[0].Value)
+        |> Outputs.apply (fun storageKeys -> Output.CreateSecret(storageKeys.Keys[0].Value))
```

Comment:
I'm surprised this works - would expect you'd need to change apply to bind?

Reply:
hah yes I think this is double nesting the output like this